tracer/test/Datadog.Trace.Tests/LambdaHandlerTests.cs

```diff
@@ -0,0 +1,35 @@
+// <copyright file="LambdaHandlerTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.IO;
+
+using Datadog.Trace.ClrProfiler.ServerlessInstrumentation;
+using FluentAssertions;
+using Xunit;
+
+namespace Datadog.Trace.Tests
+{
+    public class LambdaHandlerTests
+    {
+        [Fact]
+        public void LambdaHandlerGetters()
+        {
+            LambdaHandler handler = new LambdaHandler("mscorlib::System.Environment::ExpandEnvironmentVariables");
+            handler.GetAssembly().Should().Be("mscorlib");
+            handler.GetFullType().Should().Be("System.Environment");
+            handler.GetMethodName().Should().Be("ExpandEnvironmentVariables");
+        }
+
+        [Fact]
+        public void LambdaHandlerBuidParamTypeArray()
+        {
+            LambdaHandler handler = new LambdaHandler("mscorlib::System.Environment::ExpandEnvironmentVariables");
+            handler.ParamTypeArray.Length.Should().Be(2);
+            handler.ParamTypeArray[0].Should().Be("String");
+            handler.ParamTypeArray[1].Should().Be("System.String");
```

Comment:
This is using `Type.Name` for the return type and `Type.ToString()` for the parameters, so we end up with `String` vs `System.String`. Was this intentional?